ros2_ws/install/setup.sh

```diff
@@ -0,0 +1,49 @@
+# generated from colcon_core/shell/template/prefix_chain.sh.em
```

Comment:
Add some instruction to README so people know to source this/add to `~/.bashrc`

Reply:
task has been assigned in trello